chess/frontend/gui/src/__tests__/modules/Game/Game.test.jsx

```diff
@@ -6,6 +6,10 @@ describe("modules/Game/Game.jsx", () => {
   let game = null;
 
   beforeEach(() => {
```

Comment:
I added these lines of code to mock out the time to force it to be a certain value for the tests. If I don't do this then the snapshots will always have a new time and will fail